stories/Atom/Cards/Card-thumbnail/card-thumbnail.scss

```diff
@@ -44,4 +44,32 @@
   }
 }
 
+[dir='rtl'] {
+  .card-thumbnail__image {
+
+    &.yellow {
+      &::before {
+        @include background-image(318deg, $color-yellow, transparent 80%);
+      }
+    }
+  
+    &.red {
+      &::before {
+        @include background-image(318deg, $color-red, transparent 80%);
+      }
+    }
+  
+    &.green {
+      &::before {
+        @include background-image(318deg, $color-green, transparent 80%);
+      }
+    }
+  
+    &.azure {
+      &::before {
+        @include background-image(318deg, $color-azure, transparent 80%);
+      }
+    }
+  }
+}
```

Comment:
@Ashish-srij Can we write this as:
```
// color map
$colors: ("yellow": $color-yellow, "red": $color-red, "green": $color-green, "azure": $color-azure);

.card-thumbnail__image {
  @each $name, $color in $colors {
    &.#{$name}:before {
      @include background-image(318deg, #{$color}, transparent 80%);
    }
  }
}
```
This will compile to:
```
.card-thumbnail__image.yellow:before {
  background: linear-gradient(318deg, #ffeb00, transparent 80%);
}
.card-thumbnail__image.red:before {
  background: linear-gradient(318deg, #ee402d, transparent 80%);
}
.card-thumbnail__image.green:before {
  background: linear-gradient(318deg, #6de354, transparent 80%);
}
.card-thumbnail__image.azure:before {
  background: linear-gradient(318deg, #60d4f2, transparent 80%);
}
```

Reply:
Done